src/solver/conv_asm_implicit_gemm_gtc_fwd.cpp

```diff
@@ -477,29 +473,20 @@ static inline bool FindImplicitGemmGtcDynamicFwdKernel(const ConvolutionContext&
             continue;
         };
 
-        if(p_kernel_name != nullptr)
-            *p_kernel_name = cfg.GetKernelName();
-
-        if(p_block_size != nullptr)
-        {
-            *p_block_size = cfg.GetBlockSize();
-        };
-
-        if(p_grid_size != nullptr)
-        {
-            *p_grid_size = (gemm_m / cfg.gemm_m_per_block) * (gemm_n / cfg.gemm_n_per_block);
-        };
-
-        return true;
+        return std::make_tuple(true,
+                               cfg.GetKernelName(),
+                               cfg.GetBlockSize(),
+                               integer_divide_ceil(gemm_m, cfg.gemm_m_per_block) *
```

Comment:
[Q] This is not the same as `(gemm_m / cfg.gemm_m_per_block) * (gemm_n / cfg.gemm_n_per_block)`. Bugfix or...?

Reply:
This is to make sure have a upper-bound of grids along gemm_m and gemm_n direction. In the future we may have some kernel that not evenly divided by gemm_m_per_block but need to extended to multiply of gemm_m_per_block

Reply:
ok